just_audio_background/lib/just_audio_background.dart

```diff
@@ -285,41 +284,35 @@ class _JustAudioPlayer extends AudioPlayerPlatform {
       _playerAudioHandler.customSetShuffleOrder(request);
 
   @override
-  Future<SeekResponse> seek(SeekRequest request) async {
-    return await _playerAudioHandler.customPlayerSeek(request);
-  }
+  Future<SeekResponse> seek(SeekRequest request) =>
+      _playerAudioHandler.customPlayerSeek(request);
 
   @override
   Future<ConcatenatingInsertAllResponse> concatenatingInsertAll(
-      ConcatenatingInsertAllRequest request) async {
-    return _playerAudioHandler.customConcatenatingInsertAll(request);
-  }
+          ConcatenatingInsertAllRequest request) async =>
+      _playerAudioHandler.customConcatenatingInsertAll(request);
 
   @override
   Future<ConcatenatingRemoveRangeResponse> concatenatingRemoveRange(
-      ConcatenatingRemoveRangeRequest request) async {
-    return await _playerAudioHandler.customConcatenatingRemoveRange(request);
-  }
+          ConcatenatingRemoveRangeRequest request) =>
+      _playerAudioHandler.customConcatenatingRemoveRange(request);
 
   @override
   Future<ConcatenatingMoveResponse> concatenatingMove(
-      ConcatenatingMoveRequest request) async {
-    return await _playerAudioHandler.customConcatenatingMove(request);
-  }
+          ConcatenatingMoveRequest request) async =>
+      _playerAudioHandler.customConcatenatingMove(request);
 
   @override
   Future<SetAndroidAudioAttributesResponse> setAndroidAudioAttributes(
-      SetAndroidAudioAttributesRequest request) async {
-    return await _playerAudioHandler.customSetAndroidAudioAttributes(request);
-  }
+          SetAndroidAudioAttributesRequest request) async =>
+      _playerAudioHandler.customSetAndroidAudioAttributes(request);
 
   @override
   Future<SetAutomaticallyWaitsToMinimizeStallingResponse>
       setAutomaticallyWaitsToMinimizeStalling(
-          SetAutomaticallyWaitsToMinimizeStallingRequest request) async {
-    return await _playerAudioHandler
-        .customSetAutomaticallyWaitsToMinimizeStalling(request);
-  }
+              SetAutomaticallyWaitsToMinimizeStallingRequest request) async =>
```

Comment:
Here and elsewhere, you should remove the `async` keyword as well.

Reply:
Done